ui/css/perf.css

```diff
@@ -290,3 +290,8 @@ to { opacity: 0; }
   height: 1em;
   margin-bottom: 5px;
 }
+
+.run-detail {
+    border-bottom: 1px dotted #777;
```

Comment:
The main goal of the underline is to give some kind of hint that you can hover over the item. I might experiment with also changing the color. Light grey might be more appropriate (the # of retriggers is important, but less important than the actual number).